libstorage/src/SystemCmd.cc

```diff
@@ -256,6 +288,11 @@ SystemCmd::doExecute( string Cmd )
 		    y2error( "dup2 stderr child failed errno=%d (%s)", errno,
 			     strerror(errno));
 		    }
+		if( close( sin[1] )<0 )
+		    {
+		    y2error( "close child failed errno=%d (%s)", errno,
```

Comment:
NP (I wouldn't change already tested code for this): the original patch used "close (stdin) child failer errno", which is probably convenient for easily distinguish from the error closing sout below.